File: dataset/human/c/file_2554.c
#include<stdio.h>
int main()
{
	char s[10];
	scanf("%s",s);
	if(s[0]!=s[1]){
		if(s[1]!=s[2]){
			if(s[2]!=s[0]){
				printf("Yes");
			}else{
				printf("No");
			}
		}else{
			printf("No");
		}
	}else{
		printf("No");
	}
	return 0;
}

File: dataset/human/java/file_2554.java
import java.util.*;
public class Main{
public static void main(String[] args){
Scanner sc = new Scanner(System.in);
  String s = sc.next();
  String t = sc.next();
  char[] c1 = s.toCharArray();
  char[] c2 = t.toCharArray();
  Arrays.sort(c1);
  Arrays.sort(c2);
  c2[0] = c2[c2.length - 1];
  String S = String.valueOf(c1);
  String T = String.valueOf(c2);
  if(S.compareTo(T) < 0)System.out.println("Yes");
  else System.out.println("No");
}}
